integration_tests/tests/temporary-accommodation/manage/bedspace.cy.ts

```diff
@@ -0,0 +1,90 @@
+import premisesFactory from '../../../../server/testutils/factories/premises'
+import newRoomFactory from '../../../../server/testutils/factories/newRoom'
+import roomFactory from '../../../../server/testutils/factories/room'
+import PremisesNewPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/premisesNew'
+import Page from '../../../../cypress_shared/pages/page'
+import PremisesShowPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/premisesShow'
+import BedpsaceNewPage from '../../../../cypress_shared/pages/temporary-accommodation/manage/bedspaceNew'
+
+context('Bedspace', () => {
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubSignIn')
+    cy.task('stubAuthUser')
+  })
+
+  it('allows me to create a bedspace', () => {
+    // Given I am signed in
```

Comment:
Completely up to you but if we were starting again I think we would use the pattern established in `manage.feature` of using Gherkin syntax as opposed to just commenting it but it takes more setup/boilerplate